pytorch_lightning/strategies/parallel.py

```diff
@@ -72,6 +72,14 @@ def world_size(self) -> int:
     def is_global_zero(self) -> bool:
         return self.global_rank == 0
 
+    @property
+    def parallel_devices(self):
+        return self._parallel_devices
```

Comment:
Did you forget to rename the attribute in the `__init__`? L43

Reply:
No, it's same style as 'accelerator' and etc properties. 
Like https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/strategies/strategy.py#L44-L85

I also feel this way of lazy initialization is confusing. Should we change all of them to 
```
def __init__(accelerator):
      self._accelerator = accelerator
     
@property 
def accelerator(self):
      return self._accelerator
```
?

Reply:
Duh!

I think it's fine as it is.